.github/ISSUE_TEMPLATE/04_core_feature_request.yml

```diff
@@ -43,6 +43,16 @@ body:
         ```
     validations:
       required: true
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Priority
+      description: What is the priority of this feature with respect to your intended use?
+      options:
+        - "P0 (blocking)"
+        - "P1 (needed-urgent)"
```

Comment:
What is the difference between needed-urgent and needed? The urgency is probably represented by the time constraints section.

Reply:
That was my same thought - an aggressive time constraint seems equivalent to "urgent" and blocking would also be expressed in the dependencies section. This field does make it a bit more explicit, but my only concern is that it could be a shortcut to expressing this in the next section.

Reply:
In a plan language:

  - This blocks me; I cannot make forward progress at all without it
  - I need this as soon as possible
  - I need this, but have some timing flexibility
  - This would be cool to have at some point

My intent was to allow requesters to be explicit about their priorities rather than us inferring them; these are the priorities that I'd like to align across Core and CodeGen internally, even if we're not including them here.

Reply:
To be clear, I'm fine dropping the template change if we collectively feel like it's not helpful or redundant.

Reply:
I'm fine with it as long as it's not a substitute for the long form in the next section.